Project.toml

```diff
@@ -23,6 +23,8 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Literate = "98b081ad-f1c9-55d3-8b20-4c87d4299306"
 Lux = "b2108857-7c20-44ae-9111-449ecde12c47"
 LuxCUDA = "d0bbae9a-e099-4d5b-a835-1c6931763bda"
+NNlib = "872c559c-99b0-510c-b3b7-b6c96a88d5cd"
+NeuralClosure = "099dac27-d7f2-4047-93d5-0baee36b9c25"
```

Comment:
Hi, can you please delete NeuralClosure from `Project.toml`? We do not put it there because it breaks CI.

Reply:
I can not run `generate_data` without NeuralClosure

Reply:
This looks good to me and CI is passing.

Reply:
Yes, that is intended. That is why I put the extra instructions in `generate_data.jl`